### PR TITLE
community/stress-ng: upgrade to 0.10.04

### PR DESCRIPTION
**Pull request by prspkt**

### CODE REVIEW

**Comment by maxice8** [comment]
@oliv3r

File: community/stress-ng/APKBUILD [hunk 1]
```diff
@@ -1,34 +1,24 @@
 # Contributor: Olliver Schinagl <oliver@schinagl.nl>
 # Maintainer: Olliver Schinagl <oliver@schinagl.nl>
 pkgname=stress-ng
-pkgver=0.09.57
+pkgver=0.10.02
 pkgrel=0
-pkgdesc="stress-ng will stress test a computer system in various selectable ways."
+pkgdesc="stress-ng will stress test a computer system in various selectable ways"
```
Comment (oliv3r) [inline community/stress-ng/APKBUILD]: thanks for spotting this :)

File: community/stress-ng/APKBUILD [hunk 2]
```diff
@@ -1,34 +1,24 @@
...
 license="GPL-2.0-or-later"
 options="!check" # tests are not portable
-makedepends="
-	linux-headers
-	libaio-dev
-	libbsd-dev
-	lksctp-tools-dev
-	zlib-dev
-"
+makedepends="libaio-dev libbsd-dev linux-headers lksctp-tools-dev zlib-dev"
```
Comment (oliv3r) [inline community/stress-ng/APKBUILD]: please don't single-line this; keep it as a block, so we can more easily sort it and avoid having.
Thanks for spotting the missing sort, indeed, linux comes after the libs.

**Comment by oliv3r** [comment]
Could you commit message that the musl patch is no longer required? Bonus points for linking the patch upstream :)

File: community/stress-ng/APKBUILD (hunk 2 above)
Comment (prspkt) [reply community/stress-ng/APKBUILD]: change reverted.

**Comment by prspkt** [comment]
> Could you commit message that the musl patch is no longer required? Bonus points for linking the patch upstream :)

There is already a bullet point in the commit message about the removal. [This](https://kernel.ubuntu.com/git/cking/stress-ng.git/commit/?id=8b8c1623281c296efb9bb2ecd86a1a1ecd6b467f) is the upstream patch that fixed the issue as part of [stress-ng V0.09.58](https://kernel.ubuntu.com/git/cking/stress-ng.git/commit/?id=bf4837223e5a2cc76514ba9f529e5b3a34ccbc12) and [this](https://kernel.ubuntu.com/git/cking/stress-ng.git/commit/?id=cd8d3095b9466502209e69d39f8f2b0bf2b2e221) is the one that replaces GNU glob with scandir as part of [stress-ng V0.10.01](https://kernel.ubuntu.com/git/cking/stress-ng.git/commit/?id=55c2f78d8de627a0d0b51a143cafbdf955e62b4b).

**Pull request closed without merging**